src/Sulu/Component/Content/Document/Behavior/RouteBehavior.php

```diff
@@ -31,4 +31,32 @@ public function getTargetDocument();
      * @param object $targetDocument
      */
     public function setTargetDocument($targetDocument);
+
+    /**
+     * Returns locale.
+     *
+     * @return string
+     */
+    public function getLocale();
+
+    /**
+     * Sets locale.
+     *
+     * @param string $locale
+     */
+    public function setLocale($locale);
+
+    /**
+     * Returns history flag.
+     *
+     * @return bool
+     */
+    public function isHistory();
+
+    /**
+     * Set history flag.
+     *
+     * @param bool $history
+     */
+    public function setHistory($history);
```

Comment:
Instead of adding these methods to this behavior I would reuse the `LocaleBehavior` on the `RouteDocument` and add a separate behavior for the history. This way we can create route document being translatable and including a history, as well as simple routes without translations and history (or any of the available permutations).